stage/stage-qc-703-pls2.R

```diff
@@ -97,8 +97,6 @@ setorder(df_dt, measurementTime_PST)
 # identify runs of NAs
 df_dt[, na_run := rleid(is.na(stage))]
 
-df_dt[, qlevel := as.character(qlevel)]
```

Comment:
I think that this qlevel is being set twice but @emilyhaughton you should verify this.